prime-angular-frontend/openshift.dockerfile

```diff
@@ -34,10 +34,11 @@ COPY package.json package-lock.json ./
 COPY . .
 
 # Install Angular CLI
+RUN npm set unsafe-perm true
 RUN npm install -g @angular/cli
 # Install dependencies
 RUN npm ci
-# Add application
+# Add applicationz
```

Comment:
I know this is just a comment, but there's an extra 'z' not sure if typo or supposed to be like that :)

Reply:
Total typo! Thanks :)